### DISCUSSION TITLE
Is that possible to set variables inside editorjs content ?

### DISCUSSION THREAD

**Discussion by Aarbel** [codex-team/editor.js#2684] [Q&A]
<img width="924" alt="image" src="https://github.com/codex-team/editor.js/assets/25119847/4e54e784-f314-4a9a-9dcd-18ac528ef10a">

Is there a way to include other json blocks as "variables" inside existing editorjs content ?

**Comment by neSpecc** [comment]
Not at the moment. 

It could be a nice feature. We need to design it first. Could you explain your use case?

**Reply by Aarbel** [reply]
My use case is the following one:

- my users need to create texts with "blocks of text"
- they need to create blocks with prefilled data, ready to "fill"
- they also need to create some blocks with variables inside the text, so the variables can be filled later by automatic data, like for example an email content:

"Hello {firsName},

blabla
"
or 

"Inside the project, located "project_adress", all the tasks should be treated with the following tags: {list of tags}"
